### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by comanzanares**
Instructions added to the readme file

### CODE REVIEW

File: src/components/home/CocktailDetails.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                            </View>
+                            <Text style={{ color: 'grey' }}>{strInstructions}</Text>
+                        </ScrollView>
+                    </View>}
+            </View>
+        );
+    }
+}
+
+const styles = {
```
Comment (coreyakaplan) [inline src/components/home/CocktailDetails.js]: This styling is fine for this simple example app . However, how you would handle multiple device sizes in a production quality application? For example: iPhone 6s vs iPhone 8plus vs iPhone X? Hard coding margins and sizes will not scale between those different device sizes. Please provide a small example of how you would handle this.

File: src/actions/CocktailsActions.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+export const getCocktails = () => {
+  return (dispatch) => {
+    dispatch({ type: CHANGE_APP_PROPS, payload: true })
+    axios.get(GET_COCKTAILS_URL).then(response => {
+      let drinks = response.data.drinks
+      dispatch({ type: GET_COCKTAILS, payload: drinks })
+      dispatch({ type: CHANGE_APP_PROPS, payload: false })
+    }).catch((err) => {
+      dispatch({ type: CHANGE_APP_PROPS, payload: false })
+      console.log('error getting cocktails list', err)
```
Comment (coreyakaplan) [inline src/actions/CocktailsActions.js]: Please provide comments as to how you would provide user feedback in this case (and some details on how you would code it.). For example, if there is a reachability issue.

Comment (comanzanares) [reply src/actions/CocktailsActions.js]: I would pass a callback as a parameter in the getCocktails function and call it inside the catch statement. Inside the callback I would display a modal or a native alert to the user saying that something went wrong. Maybe also give the user the possibility of retrying the request or prompting another screen. 
So something like this:
(where I call the action):
```
this.props.getCocktails(this.onGetCocktailsError)
onGetCockailsError(err){
Alert.alert('An error occurred while fetching data, please try again later', err)
}
```
inside the catch statement: 
```
export const getCocktails = (errorCallback) => {
...
.catch((err) => {
errorCallback(err)
})
```

File: src/components/home/CocktailDetails.js (hunk 1 above)
Comment (comanzanares) [reply src/components/home/CocktailDetails.js]: I could use get the dimensions of the viewport and work with scales
const { width, height } = Dimensions.get('window')
...
 marginLeft: width * 0.8,
marginTop: height * 0.5

**Pull request closed without merging**